storage/badgerdb/badgerdb.go

```diff
@@ -208,6 +208,14 @@ func (s *DB) Destroy() error {
 	return err
 }
 
+// DestroyClosed removes the already closed storage medium stored data
+func (s *DB) DestroyClosed() error {
+	s.dbClosed <- struct{}{}
+	err := os.RemoveAll(s.path)
```

Comment:
you can return directly os.RemoveAll(s.path)

Reply:
done